### ISSUE TITLE
GRUB menu doesn't show up post-install while dual booting with Windows 10

### ISSUE THREAD

**Issue by daesteros** [blend-os/blendOS#107]
After rebooting, windows boot manager boots the system as it is first in order(and blend doesn't show up at all). Tried enabling both uefi-only and uefi and legacy boot in bios to no avail. There's an issue with /boot/EFI partition not being created from Jade on there, so maybe merge it with this?

**Comment by ThinkSalat** [comment]
Same thing happening for me.

**Comment by CtheCondor** [comment]
Did you do manual partitioning? I couldn't get the manual to work on my system either at all and ended up just going automatically and that worked to install and show up in the boot menu. If you're using a single drive going automatic is not feasible I know but maybe the maintainers can figure out what they are doing well in the auto partitioning that is not working well with that Manual options.

**Comment by daesteros** [comment]
> Did you do manual partitioning? I couldn't get the manual to work on my system either at all and ended up just going automatically and that worked to install and show up in the boot menu. If you're using a single drive going automatic is not feasible I know but maybe the maintainers can figure out what they are doing well in the auto partitioning that is not working well with that Manual options. 

Yeah, I did manual on single disk (dual boot with Windows). Will try to test auto on a spare laptop, thanks

**Comment by 7A9Oo** [comment]
> > Did you do manual partitioning? I couldn't get the manual to work on my system either at all and ended up just going automatically and that worked to install and show up in the boot menu. If you're using a single drive going automatic is not feasible I know but maybe the maintainers can figure out what they are doing well in the auto partitioning that is not working well with that Manual options.
> 
> Yeah, I did manual on single disk (dual boot with Windows). Will try to test auto on a spare laptop, thanks

if you try to dual booting with windows just choose efi partition of windows as bootable without formatting
and the root
and after install i think you got grub with dual boot
if not check the documentation
https://docs.blendos.co/docs/installation/normal-pc#appendicies
